server/index.py

```diff
@@ -13,10 +15,12 @@ def status():
     return "it's up!"
 
 
-@app.route('/test-model', methods=['GET'])
-def get_text():
-    """Use pretrained model to generate text by the requested parameters."""
+@app.route('/lstm_trump_tweets_model', methods=['GET'])
```

Comment:
I'd prefer something like `/model/lstm/trump`, or possibly specify `lstm` and `trump_tweets` as parameters/in the JSON body?

Reply:
done!